### PR TITLE
Issue 38: TOC added to README.md

### PR DESCRIPTION
**Pull request by Aparnarr**
Signed-off-by: Aparna <Aparna.Rr@emc.com>

*Change log description
1. TOC added
2. Included the following sections:
     a. Run operator locally added
     b. Releases added.

*Purpose of the change
Fixes #38, Closes #43

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,18 +31,18 @@ The Pravega operator manages Pravega clusters deployed to Kubernetes and automat
 - [x] Resize cluster
 - [ ] Rolling upgrades
 
-> Note that unchecked features are in the roadmap but not available yet.
+> Note that the unchecked features are in the roadmap but not available yet.
```
Comment (Aparnarr) [inline README.md]: Note that the.......`the`  will be removed in the next revision

**Comment by Aparnarr** [comment]
@adrianmo 
hope i have addressed ur comments

File: README.md [hunk 2]
```diff
@@ -1,9 +1,29 @@
+
```
Comment (adrianmo) [inline README.md]: Remove these blank lines.

File: README.md [hunk 3]
```diff
@@ -1,9 +1,29 @@
...
+
 # Pravega Operator
 
 ### Project status: alpha
 
 The project is currently alpha. While no breaking API changes are currently planned, we reserve the right to address bugs and change the API before the project is declared stable.
 
+# Table of Contents
```
Comment (adrianmo) [inline README.md]: Change to a level 2 title (two `#`).

File: README.md [hunk 4]
```diff
@@ -1,9 +1,29 @@
...
+
+ * [Overview](#overview)
+ * [Requirements   ](#requirements)
```
Comment (adrianmo) [inline README.md]: Remove these trailing spaces in all items.

File: README.md [hunk 5]
```diff
@@ -1,9 +1,29 @@
...
+ * **Usage**
```
Comment (adrianmo) [inline README.md]: Remove bold and link it to the "Usage" section.

File: README.md [hunk 6]
```diff
@@ -1,9 +1,29 @@
...
+# Table of Contents
+
+ * [Overview](#overview)
+ * [Requirements   ](#requirements)
+ * **Usage**    
+    * [Installation of the Operator    ](#install-the-operator)
+    * [Deploy a sample Pravega Cluster   ](#deploy-a-sample-pravega-cluster)
+    * [Uninstall the Pravega Cluster    ](#uninstall-the-pravega-cluster)
+    * [Uninstall the Operator    ](#uninstall-the-operator)
+ * **Development**
```
Comment (adrianmo) [inline README.md]: Remove bold and link it to the "Development" section.

File: README.md [hunk 7]
```diff
@@ -1,9 +1,29 @@
...
+    * [Build the Operator Image    ](#build-the-operator-image)
+    * [Tier 2: Google Filestore Storage    ](#using-google-filestore-storage-as-tier-2)
+    * [Tuning Pravega Configurations    ](#tuning-pravega-configuration)
+    * [Installation on GKE    ](#installation-on-google-kubernetes-engine)
+    * [Direct Access to Cluster    ](#direct-access-to-the-cluster)
+    * [Run the Operator Locally   ](#run-the-operator-locally)
+* [Releases](#releases)
+
 ### Overview
```
Comment (adrianmo) [inline README.md]: Change it to level 2 title (two `#`).

File: README.md [hunk 8]
```diff
@@ -17,15 +31,12 @@ The Pravega operator manages Pravega clusters deployed to Kubernetes and automat
 > Note that unchecked features are in the roadmap but not available yet.
 
 ## Requirements
-
```
Comment (adrianmo) [inline README.md]: Why did you remove all those lines? Please restore them.

File: README.md (hunk 4 above)
Comment (adrianmo) [reply README.md]: @Aparnarr remove the trailing spaces from the ToC items.
